engine_modules/mdc/admin.py

```diff
@@ -1,14 +0,0 @@
-from django.contrib import admin
-
-from engine_modules.mdc.models import MDCVoteSession, MDCVoteOrder
-
-
-class MDCVoteSessionAdmin(admin.ModelAdmin):
```

Comment:
ahaha ;) next time you should try to use `git mv` instead to keep the history ;)